frontend/src/components/Region.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
```

Comment:
@syedomair0 It's good that you moved Region fields into a new component. But you have to revert the textbox styles to its previous version or make sure all fields are consistent in their size.
![image](https://github.com/UAVLab-SLU/DRV_public/assets/73876259/a6c9a007-568d-4944-b8ac-159c974fc42c)